gulpfile.js

```diff
@@ -130,6 +130,10 @@ gulp.task('img', function() {
   return gulp.src(config.src.img).pipe(gulp.dest(config.dest.img));
 });
 
+gulp.task('templatedata', function() {
+  return gulp.src(config.src.templateData).pipe(gulp.dest(config.dest.templateData));
```

Comment:
JSON endpoints containing products are stored in templates/data/*.json. This should be copied over to dist.